.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Issue outline
+
+## Context
+
+Repository that you are referring to
+* GitHub Repository:
+
+Please provide any additional context around the issue below.
+
+## Expected Behavior
+
+Please describe the behavior you are expecting
+
+## Current Behavior
+
+What is the current behavior?
+
+
+# Bugs only
+## Failure Information
+
+Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.
+
+### Steps to Reproduce
+
+Please provide detailed steps for reproducing the issue.
+
+1. step 1
```

Comment:
I quite like that the govuk-frontend issue template for bug reports has this text in a html comment

>  Please fill in as much of the template below as you’re able to. If you're unsure whether the issue already exists or how to fill in the template, open an issue anyway. Our team will help you to complete the rest.
> 
>  Your issue might already exist. If so, add a comment to the existing issue instead of creating a new one. You can find existing issues here:
>  - an existing Github issue: https://github.com/alphagov/govuk-frontend/issues
>  - our known validation error/warning list: https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851

https://raw.githubusercontent.com/alphagov/govuk-frontend/main/.github/ISSUE_TEMPLATE/bug-report.md

specifically I like this, and wonder if we could add it too: 
> Please fill in as much of the template below as you’re able to. If you're unsure whether the issue already exists or how to fill in the template, open an issue anyway. Our team will help you to complete the rest.

Reply:
or we could even pilfer more of their template verbatim, just noticed they have some comments in each section too 🤷‍♂️